appvia/kore
# allow deletion of clusters with kore DNS zones

## Summary

Will allow for deleting clusters that have DNS domains.

**Which issue(s) this PR resolves**:

Resolves #1356 

## Changes

So the right annotations have been added to plans and objects so that domains created are marked with the annotation `kore.appvia.io/system`

## Testing

1. Create an organisation domain in Configure / DNS and ensure it's delegation is complete.
2. Create or wait for a cluster to get it's child domains created
3. Try and delete the cluster

No errors or warnings should occur about DNS domains when starting the delete process or during (checking the details during the delete progress e.g.:

![image](https://user-images.githubusercontent.com/2500933/93981766-47291900-fd78-11ea-81c3-e2a3b52eb8d1.png)
